web/html/src/utils/functions.ts

```diff
@@ -119,14 +119,14 @@ function capitalize(str: string): string {
 
 function generatePassword(): string {
   const length = Math.floor(Math.random() * 10) + 15;
+  // See https://stackoverflow.com/a/68617567/1470607
   const charset = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789:-_";
+  const array = new Uint32Array(charset.length);
+  window.crypto.getRandomValues(array);
+
   let password = "";
-  if (window.crypto && window.crypto.getRandomValues) {
```

Comment:
Can we ensure that `window.crypto` is alsways defined or should we keep the check?
I'd guess the answer is yes but it isn't immediately obvious to me.

Reply:
It's available on all the browsers we support, I think this check was originally added quite a while ago.